migrations/4_deploy_OWL_airdrop.js

```diff
@@ -16,6 +22,7 @@ module.exports = function (deployer, network, accounts) {
     network,
     accounts,
     web3,
-    gnoLockEndTime
+    gnoLockEndTime,
+    multiplier: process.env.MULT
```

Comment:
I would set a name more semantic, like OWL_PER_GNO, also, I think is best if we assert that it was provided.